App/Redux/ProcessingImagesRedux.ts

```diff
@@ -202,6 +208,20 @@ export function reducer (state: ProcessingImagesState = initialState, action: Pr
       })
       return { ...state, images }
     }
+    case getType(actions.uploadError): {
```

Comment:
We need to track errors for individual uploads now, this special action is for that. The other `error` action is what's used for higher level error around preparing the image, adding it to a thread, etc.